client/desktop/lib/config/index.js

```diff
@@ -1,4 +1,4 @@
-const pkg = require( '../../../package.json' );
+const pkg = require( '../../../../desktop/package.json' );
```

Comment:
Unrelated, but as I was testing I caught an error that didn't resolve the desktop config correctly with respect to the `package.json` in `desktop/package.json` - fixed the import misspecification.